examples/extensions-collection/collection.json

```diff
@@ -64,5 +64,5 @@
       ]
     }
   },
-  "license": "PDDL-1.0"
```

Comment:
should we keep this as an example of using an SPDX license identifier?

Reply:
I think there are other examples covering this case (if not, let me know). 
This was changed as there was a link in the document that didn't point to PDDL and thus was inconsistent.